packages/docs/src/components/layout/HeaderFullWidth.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CMSLogoIcon from '../icons/CMSLogo';
+import SearchForm from '../content/SearchForm';
+
+const HeaderFullWidth = () => {
+  return (
+    <div className="ds-l-row ds-u-margin--0 ds-u-padding--2 ds-u-padding-left--0 ds-u-border-bottom--2">
+      <div className="ds-u-display--flex ds-l-col--6 ds-u-align-items--center ds-u-padding-left--0">
+        <CMSLogoIcon />
+        <h1 className="ds-u-padding-left--1 ds-text-heading--2xl ds-u-margin--0">Design System</h1>
+      </div>
+      <SearchForm className="ds-u-display--flex ds-l-col--6 ds-u-align-items--center ds-u-justify-content--end" />
```

Comment:
This layout doesn't really work for mobile. Should we hide the search form on mobile and stick another one in the mobile menu that is hidden on desktop?

Reply:
Yeah we should show the search link on mobile in the header maybe?

Reply:
Everything besides the mobile view of the header LGTM. Mobile view used to look like this:

<img width="377" alt="Screenshot 2023-05-16 at 9 44 29 AM" src="https://github.com/CMSgov/design-system/assets/7595652/985e833a-1f26-4e97-bd91-8ce97f83f188">

and now it looks like this:

<img width="384" alt="Screenshot 2023-05-16 at 9 44 38 AM" src="https://github.com/CMSgov/design-system/assets/7595652/469ba4c9-f180-4c54-b6da-ffc6f76ed1fe">

The spacing on the page content looks better, but we're missing the site heading, and the padding is off.

Reply:
@line47 , how would you feel about putting the menu button the right side like we do on healthcare.gov? That would allow the site title to left-align with the text on the page if we got those margins right.

Reply:
@forrestbaer, I see it moved into the mobile menu, but the header is still blank on mobile when it should probably say "CMS Design System" like it did before